.ci-operator.yaml

```diff
@@ -2,4 +2,4 @@
 build_root_image:
   namespace: openshift
   name: release
-  tag: rhel-9-release-golang-1.20-openshift-4.14
+  tag: rhel-9-release-golang-1.20-openshift-4.15
```

Comment:
Can we do this later in a separate PR? We want to do this in the Dockerfiles too but IMO it's not needed for this release

Reply:
Good point, I don't anticipate any issues as it has been used already https://github.com/openshift/windows-machine-config-operator/blob/release-4.15/build/Dockerfile.ci#L6 

Happy to drop it if the team decides otherwise.

Reply:
Wait my local branch was out of date, I didn't realize we've already bumped to using 4.15 in the dockerfiles. In that case I am fine with leaving this change in